application/core/QuestionTypes/MultipleNumericalQuestion/RenderMultipleNumerical.php

```diff
@@ -318,6 +318,23 @@ public function render($sCoreClasses = '')
             $dynamicTemplate = "/survey/questions/answer/multiplenumeric/rows/dynamic_slider.twig";
         }
 
+        $displaytotal     = false;
+        $equals_num_value = false;
+        if (trim($this->getQuestionAttribute('equals_num_value')) != ''
+        || trim($this->getQuestionAttribute('min_num_value')) != ''
+        || trim($this->getQuestionAttribute('max_num_value')) != ''
+        ) {
+            $qinfo = LimeExpressionManager::GetQuestionStatus($this->oQuestion->qid);
+
+            $sumRemainingEqn = LimeExpressionManager::ProcessString('{'.$qinfo['sumRemainingEqn'].'}', $this->oQuestion->qid);
```

Comment:
Maybe better to use https://github.com/LimeSurvey/LimeSurvey/blob/720c3dac0790c78d4623de0e448b684ea1f04d8f/application/core/LS_Twig_Extension.php#L513 in twig file ?

Then : a QuestionTheme can check if it's fixed number to do «««some action»»» for example ?

Unsure on the best way : but i like to send raw attributes to QuestionTheme , think it's better for adaptation.